Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props => {
+    return (
+        <h3>
+            {props.headerTitleProps.h1Text}
+            <span>{props.headerTitleProps.spanText}</span>
```

Comment:
As before: best practice would be to hard code the content on the component, and reference the component in the container (to save you from passing around too many props, and making the code a little clearer.